src/bin/server.rs

```diff
@@ -333,12 +392,31 @@ fn main() {
         process::exit(1);
     }
 
-    let (addr, key_seed) = load_config(&args.nth(1).unwrap());
+    let (addr, key_seed, batch_size) = load_config(&args.nth(1).unwrap());
     let (mut ephemeral_key, cert_bytes) = make_key_and_cert(&key_seed);
 
     info!("Server listening on {}", addr);
 
-    polling_loop(&addr, &mut ephemeral_key, &cert_bytes);
+    if env::var("BENCH").is_ok()  {
```

Comment:
Seems a bit out-of-place, the benchmark code popping-up in the middle of the server's `main`. Can it move elsewhere?